README.md

```diff
@@ -87,22 +87,29 @@ The tee-evidence formats depend on the TEE and are typically defined by the each
 Hash of `nonce` and `tee-pubkey` should be embedded in report/quote in `tee-evidence`, so they can be signed by HW-TEE.
 This mechanism ensures the freshness of Evidence and the authenticity of `tee-pubkey`.
 
-### Attestation result format:
+### Attestation Results Token:
+
+Attestation results token is a [JSON Web Token](https://datatracker.ietf.org/doc/html/rfc7519) which contains the parsed evidence claims such as TCB status.
+
+Claims format of the attestation results token is:
 
 ```json
 {
-    "tee": $tee_type,
-    "allow": true,
-    "output": {
-        "verifier_output": $verifier_output,
-        "policy_engine_output": $opa_output
```

Comment:
Do we need to mention that the original opa_output will be raised as an error information to the client side?